entity-registry/src/main/java/com/linkedin/metadata/aspect/patch/builder/DashboardInfoPatchBuilder.java

```diff
@@ -52,6 +55,22 @@ public DashboardInfoPatchBuilder removeDatasetEdge(@Nonnull DatasetUrn urn) {
     return this;
   }
 
+  // Simplified with just Urn
```

Comment:
Let's add tests for these

Reply:
I found these tests for `DashboardInfoPatchBuilder`

https://github.com/datahub-project/datahub/blob/79aa40f1e69632ab6e0d63c1c9554a682a99424c/metadata-integration/java/datahub-client/src/test/java/datahub/client/patch/PatchTest.java#L619

I will add it there, even if they are all `@Ignore`.

Reply:
addressed in commit 49e0dc2738303ae381c3e2b3be1e3436649c06df